src/workspacer.Shared/Layout/GoldenLayoutEngine.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace workspacer
+{
+    public class GoldenLayoutEngine : ILayoutEngine
+    {
+        private readonly int _numInPrimary;
+        private readonly double _primaryPercent;
+        
+        private int _numInPrimaryOffset = 0;
+        private double _primaryPercentOffset = 0;
+        
+        public GoldenLayoutEngine() : this(1, 0.618) { }
+
+        public GoldenLayoutEngine(int numInPrimary, double primaryPercent)
+        {
+            _numInPrimary = numInPrimary;
+            _primaryPercent = primaryPercent;            
+        }
+        
+        public string Name => "Golden";
```

Comment:
this is a minor nitpick, but can this be `golden` instead of `Golden`? so that it matches the other engines

Reply:
Hi! Apologize to didn't reply to your suggestion! Of course, I'll work on that and improve it next weeks. 
Many thanks for your work!